### PR TITLE
Add view for CIL file

### PR DESCRIPTION
**Pull request by stilscher**
This PR adds a toggle button to switch between a source file and the CIL file. PR https://github.com/goblint/analyzer/pull/1389 contains the respective changes in Goblint to marshall the cil file.

Builds on #42

### CODE REVIEW

File: src/App.re [hunk 1]
```diff
@@ -134,6 +135,10 @@ let init = (solver, spec, config, meta, cil, analyses, warnings, stats, file_loc
     | _ => raise(InitFailed("Failed to load file path table"))
     };
 
+  let cilfilename = "file.cil";
+  let oc = Stdlib.open_out(cilfilename);
+  dumpFile(defaultCilPrinter, oc, cilfilename, cil);
```
Comment (sim642) [inline src/App.re:140]: It should be possible to use `printGlobal` (per-global) to directly invoke `defaultCilPrinter` to produce a `Pretty.doc` which can be converted to string with `Pretty.sprint`.

But this is probably simpler for now.

**Review by michael-schwarz — APPROVED** [review]
Nice! It would also be great if the warnings etc were also shown in this view!

**Comment by sim642** [comment]
> It would also be great if the warnings etc were also shown in this view!

That's not so straightforward though: for any Goblint CFG node (or CIL `stmt`) we only know the location where it was parsed from, but not the location where it would get printed by any particular `cilPrinter` (either the default or the clean one).

**Comment by stilscher** [comment]
I also thought about that, but I think this can also be solved in a separate PR.

**Pull request merged**